src/Symfony/Component/Notifier/Bridge/Telnyx/composer.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "symfony/telnyx-notifier",
+    "type": "symfony-bridge",
+    "description": "Symfony Telnyx Notifier Bridge",
+    "keywords": ["sms", "telnyx", "notifier"],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Fabien Potencier",
+            "email": "fabien@symfony.com"
+        },
```

Comment:
You should add yourself as an author here

Reply:
Added